imqsauth/http.go

```diff
@@ -776,6 +800,31 @@ func httpHandlerUpdateUser(central *ImqsCentral, w http.ResponseWriter, r *httpR
 	}
 }
 
+func httpHandlerUnlockUser(central *ImqsCentral, w http.ResponseWriter, r *httpRequest) {
+	username := strings.TrimSpace(r.http.URL.Query().Get("username"))
+
+	userId, err := getUserId(r)
+	if err != nil {
+		authaus.HttpSendTxt(w, http.StatusBadRequest, err.Error())
+		return
+	}
+
+	// Get the userId of the logged-in user
```

Comment:
BUT.. then why do add this comment "of the logged-in user"?
That doesn't make any sense?

Reply:
That's just a bad comment, my doing